app/scripts/controllers/edit/buildConfig.js

```diff
@@ -265,6 +265,8 @@ angular.module('openshiftConsole')
                 var secretsByType = SecretsService.groupSecretsByType(secrets);
                 var secretNamesByType =_.mapValues(secretsByType, function(secrets) {return _.map(secrets, 'metadata.name');});
                 $scope.webhookSecrets = SecretsService.groupSecretsByType(secrets).webhook;
+                // Add empty option to the webhook trigger secrets
```

Comment:
Consider switching this comment to `why` rather than `what`.  IE, I see the `.unshift('')`, so I can tell it's adding an option, but I'm guessing there was a reason you wanted to leave this comment.